src/lib/swapper/swappers/OsmosisSwapper/utils/helpers.ts

```diff
@@ -149,9 +149,12 @@ export const getRateInfo = async (
 ): Promise<Result<PoolRateInfo, SwapErrorRight>> => {
   const sellAmountOrDefault = sellAmount === '0' ? '1' : sellAmount
   const maybePool = await findPool(sellAsset, buyAsset, osmoUrl)
-  return maybePool.andThen(({ pool, sellAssetIndex, buyAssetIndex }) =>
-    Ok(getPoolRateInfo(sellAmountOrDefault, pool, sellAssetIndex, buyAssetIndex)),
-  )
+  return maybePool.match({
+    ok: ({ pool, sellAssetIndex, buyAssetIndex }) => {
+      return getPoolRateInfo(sellAmountOrDefault, pool, sellAssetIndex, buyAssetIndex)
```

Comment:
No need to wrap, as `getPoolRateInfo` returns `Result<PoolRateInfo, SwapErrorRight>`.